qiskit/dagcircuit/dagcircuit.py

```diff
@@ -526,10 +526,8 @@ def _add_op_node(self, op, qargs, cargs):
         self._increment_op(op)
         return node_index
 
-    @deprecate_function(
-        "The DAGCircuit._copy_circuit_metadata method is deprecated as of 0.20.0. It will be "
-        "removed no earlier than 3 months after the release date. You should use the "
-        "DAGCircuit.copy_empty_like method instead, which acts identically.",
+    @deprecate_func(
+        additional_msg="Instead, use :meth:`~copy_empty_like()`, which acts identically.",
```

Comment:
Does it work with `()`?
```suggestion
        additional_msg="Instead, use :meth:`~copy_empty_like`, which acts identically.",
```

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-func

> The role text needs not include trailing parentheses to enhance readability; they will be added automatically by Sphinx if the [add_function_parentheses](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-add_function_parentheses) config value is True (the default).